ci-operator/step-registry/ipi/conf/nutanix/manual-creds/ipi-conf-nutanix-manual-creds-commands.sh

```diff
@@ -6,17 +6,42 @@ set -o pipefail
 
 source "${SHARED_DIR}/nutanix_context.sh"
 
-credobj="[{\"type\":\"basic_auth\",\"data\":{\"prismCentral\":{\"username\":\"${NUTANIX_USERNAME}\",\"password\":\"${NUTANIX_PASSWORD}\"},\"prismElements\":null}}]"
-credentials=$( echo -n "${credobj}" | base64 -w 0 )
-
-# machine-api credentials manifest
-cat > "${SHARED_DIR}/manifest_openshift-machine-api-nutanix-credentials-credentials.yaml" << EOF
-apiVersion: v1
-kind: Secret
-metadata:
-   name: nutanix-credentials
-   namespace: openshift-machine-api
-type: Opaque
-data:
-  credentials: ${credentials}
+CR_DIR="/tmp/credentials_request"
+mkdir -p "${CR_DIR}"
+
+MANIFEST_PREFIX="${SHARED_DIR}/manifests"
```

Comment:
Please change "/manifests" to "/manifest", the line should be:
MANIFEST_PREFIX="${SHARED_DIR}/manifest"

Look at https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L364:
find "${SHARED_DIR}" \( -name "manifest_*.yml" -o -name "manifest_*.yaml" \)